fuzzers/frida_libpng/src/fuzzer.rs

```diff
@@ -140,7 +140,7 @@ unsafe fn fuzz(options: &FuzzerOptions) -> Result<(), Error> {
                         // RNG
                         StdRand::with_seed(current_nanos()),
                         // Corpus that will be evolved, we keep it in memory for performance
-                        CachedOnDiskCorpus::no_meta(PathBuf::from("./corpus_discovered"), 64)
+                        CachedOnDiskCorpus::no_meta(PathBuf::from("./corpus_discovered"), 4)
```

Comment:
For an example fuzzer this might be a bit small?

Reply:
yeah that was for debugging
i forgot to remove